data_models/README.md

```diff
@@ -137,7 +138,18 @@ The fields will be used for normalizing the county case and death tabulations, a
     }
 ```
 
-5. __Hospitalization Data__
+6. __Inmate Data__
+
```

Comment:
You could update this part to say that data collection is pending resolution of #108.